MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/RecommendationsView/RecommendationsView.swift

```diff
@@ -13,6 +13,8 @@ class RecommendationsView: UIView {
     var collectionView: UICollectionView!
     lazy var dataSource = makeDataSource()
     
+    var selectedMovie: ((Int) -> ())!
```

Comment:
You can use [passthroughSubject](https://developer.apple.com/documentation/combine/passthroughsubject) and then just push new values from the `didSelectItemAt` and then just subscribe to it.